docker/Dockerfile-dev

```diff
@@ -40,7 +38,7 @@ ADD collect_static.sh /
 ADD startup.sh /
 ADD test.sh /
 ADD manage /usr/local/bin/manage
-ADD manage /usr/local/bin/django-admin
+ADD django-admin /usr/local/bin/django-admin
```

Comment:
WHAT?  WHAT WHAT WHAT?
I thought after all that discussion that we had picked manage???